pkgs/development/python-modules/libversion/default.nix

```diff
@@ -28,6 +34,15 @@ buildPythonPackage rec {
     libversion
   ];
 
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  preCheck = ''
```

Comment:
```suggestion
  postBuild = ''
```

Reply:
I'm not sure tbh but I'm more comfortable with doing that after installation.

Reply:
all good then